controllers/ordersController.js

```diff
@@ -32,4 +32,21 @@ ordersController.buildDelete = async function(req, res){
     })
 }
 
+ordersController.createOrder = async function(req, res){
+    const { product_name, measurement_name, size_type, fastener_type, color_name, mesh_type, mirage_build_out, mirage_3500_handle } = req.body
+    console.log('Creating order')
+    try {
+        const order = await ordersModel.createOrder(
+            product_name,
+            measurement_name,
+            size_type,
+            fastener_type,
+            color_name,
+            mesh_type,
+            mirage_build_out,
+            mirage_3500_handle
+        )
+    }
```

Comment:
You need an except block here.

Reply:
Yes, you're right. I should've commented it out or not synced it until it was complete/not throwing an error. I'm glad you noticed.